### PR TITLE
deployment config schema change

### PR DESCRIPTION
**Pull request by kushalmalani**
astronomer/issues#1419

### CODE REVIEW

File: cmd/deployment.go [hunk 1]
```diff
@@ -211,18 +211,18 @@ func deploymentCreate(cmd *cobra.Command, args []string, client *houston.Client,
 	// Silence Usage as we have now validated command input
 	cmd.SilenceUsage = true
 
-	deploymentConfig := make(map[string]string)
+	var executorType string;
```
Comment (andriisoldatenko) [inline cmd/deployment.go:214]:
```suggestion
	var executorType string
```

File: deployment/deployment.go [hunk 2]
```diff
@@ -45,8 +45,9 @@ func checkManualReleaseNames(client *houston.Client) bool {
 }
 
 // Create airflow deployment
-func Create(label, ws, releaseName, cloudRole string, deploymentConfig map[string]string, client *houston.Client, out io.Writer) error {
-	vars := map[string]interface{}{"label": label, "workspaceId": ws, "config": deploymentConfig, "cloudRole": cloudRole}
+func Create(label, ws, releaseName, cloudRole string, executor string, client *houston.Client, out io.Writer) error {
+
```
Comment (adam2k) [inline deployment/deployment.go:49]: Did the linter add this extra line?  If not can you remove it?

File: deployment/deployment_test.go [hunk 3]
```diff
@@ -115,10 +115,8 @@ func TestCreate(t *testing.T) {
   "data": {
     "createDeployment": {
       "id": "ckbv818oa00r107606ywhoqtw",
-      "config": {
-        "executor": "CeleryExecutor"
-      },
-      "urls": [
+			"executor": "CeleryExecutor",
```
Comment (adam2k) [inline deployment/deployment_test.go:118]: Is this the correct indention?  Did the linter do this too?

File: deployment/deployment_test.go [hunk 4]
```diff
@@ -259,9 +253,7 @@ func TestUpdate(t *testing.T) {
   "data": {
     "updateDeployment": {
       "id": "ckbv801t300qh0760pck7ea0c",
-      "config": {
-        "executor": "CeleryExecutor"
-      },
+			"executor": "CeleryExecutor",
```
Comment (adam2k) [inline deployment/deployment_test.go:256]: Same indention comment here

File: deployment/deployment_test.go (hunk 3 above)
Comment (kushalmalani) [reply deployment/deployment_test.go:118]: Not sure why it looks different on the github UI. It looks fine on VS.

File: deployment/deployment_test.go (hunk 4 above)
Comment (andriisoldatenko) [reply deployment/deployment_test.go:257]: @kushalmalani can you run go fmt?

Comment (kushalmalani) [reply deployment/deployment_test.go:257]: I did that...no format changes locally

**Pull request merged**